server/sonar-server/src/main/java/org/sonar/server/component/ws/ShowAction.java

```diff
@@ -70,7 +70,9 @@ public void define(WebService.NewController context) {
       .setSince("5.4")
       .setChangelog(
         new Change("6.4", "Analysis date has been added to the response"),
-        new Change("6.4", "The field 'id' is deprecated in the response"))
+        new Change("6.4", "The field 'id' is deprecated in the response"),
+        new Change("6.4", "The 'visibility' field is added")
```

Comment:
response sample of api/components/show should be updated, I don't see it in this commit ("SONAR-9099 add visibility flag to response of api/components/show")